node.gyp

```diff
@@ -957,7 +937,7 @@
           'type': 'none',
         }],
       ],
-    }
+    }, # cctest
```

Comment:
These make this quite a bit nicer to read, thanks :+1:

Reply:
Especially in editors that know how to fold
![image](https://user-images.githubusercontent.com/96947/46260036-11196d00-c4af-11e8-9b86-52ea0f9c1be6.png)